test/TestServerProjectsLowLevel/llc_update1/Custom/ParamsClient.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+
+namespace llc_update1_LowLevel
+{
+    /// <summary> The Params service client. </summary>
+    [CodeGenSuppress("PostParametersAsync", typeof(RequestContent), typeof(RequestContext))]
+    [CodeGenSuppress("PostParameters", typeof(RequestContent), typeof(RequestContext))]
+    [CodeGenSuppress("CreatePostParametersRequest", typeof(RequestContent), typeof(RequestContext))]
+    public partial class ParamsClient
```

Comment:
@fengzhou-msft, I see you've added customization here.  Is the idea that once we've introduced "state in autorest" we'll come back and update this test to handle it fully generated?

Reply:
yes, that's the plan.